src/framework/ui/select/select.component.tsx

```diff
@@ -137,6 +143,14 @@ interface State {
  *
  * @overview-example SelectWithGroups
  *
+ * @overview-example SelectPreselectedInline
```

Comment:
let's leave only this example of a preselected value

Reply:
I guess we must show at least 2 examples, "reference" and "inline"